pkg/collect/longhorn.go

```diff
@@ -307,6 +289,29 @@ func (c *CollectLonghorn) Collect(progressChan chan<- interface{}) (CollectorRes
 	return output, nil
 }
 
+func (c *CollectLonghorn) collectLonghornLogs(namespace string, results CollectorResult, progressChan chan<- interface{}, ) error {
+		// logs of all pods in namespace
+		logsCollectorSpec := &troubleshootv1beta2.Logs{
+			Selector:  []string{""},
+			Name:      GetLonghornLogsDirectory(namespace), // Logs (symlinks) will be stored in this directory
```

Comment:
This it where the fix is

We tell the logs collector to store logs where longhorn expects them. No need of trying to copy files from some other directory using `copyResult()`

Reply:
> 

Good question. Analysers will follow symlinks when reading results

Here is an analysis result

<img width="1230" alt="image" src="https://user-images.githubusercontent.com/681087/205347910-ffdb674b-79c8-422d-895d-cd863c74195c.png">